<launch>
    <rosparam file="$(find wrp_sdk_periph)/config/imu_sensor_node_config.yaml" />
    <node pkg="wrp_sdk_periph" name="imu_sensor_node" type="imu_sensor_node" output="screen"/>
</launch>